crates/mpz-ole-core/src/core/mod.rs

```diff
@@ -0,0 +1,123 @@
+//! This implementation is based on the COPEe protocol from <https://eprint.iacr.org/2016/505> page 10
+//! with the following modification:
+//!
+//! - The `Initialize` stage is instantiated using random OT rather than chosen-input OT.
+//! - The `Extend` stage can only be called once, since our goal is to implement oblivious linear
+//!   function evaluation (OLE) rather than vector OLE (VOLE) (which means that we do not use PRGs).                                                  
+//!                                                                                       
```

Comment:
```suggestion
//! - The evaluated function is f(b)=a*b+x rather than f(b)=a*b-x.                                                  
//!                                                                                       
```